serverless/event_sources/knative-event-sources.adoc

```diff
@@ -1,25 +1,23 @@
 include::modules/serverless-document-attributes.adoc[]
 [id="knative-event-sources"]
-= Getting started with event sources
+= Understanding event sources
 include::modules/common-attributes.adoc[]
 :context: knative-event-sources
 
 toc::[]
 
-An _event source_ is an object that links an event producer with an event _sink_, or consumer. A sink can be a Knative service, channel, or broker that receives events from an event source.
+A Knative _event source_ can be any Kubernetes object that generates or imports CloudEvents, and relays those events to another endpoint, known as a xref:../../serverless/knative_eventing/serverless-event-sinks.adoc#serverless-event-sinks[_sink_]. Sourcing events is critical to developing a distributed system that reacts to events.
```

Comment:
```suggestion
A Knative _event source_ can be any Kubernetes object that generates or imports CloudEvents. Event sources relay those events to an endpoint, which is known as a xref:../../serverless/knative_eventing/serverless-event-sinks.adoc#serverless-event-sinks[_sink_]. Sourcing events is critical to developing a distributed system that reacts to events.
```
Just a thought about breaking this up.

Reply:
I kind of like this the way it is if that's OK, I think it's a better flow than breaking it up.